### DISCUSSION TITLE
make generators use the same shape and color as the generated thing

### DISCUSSION THREAD

**Discussion by nobody324324123** [Taureon/aps-plus-plus#218] [Suggestions]
title

**Comment by 0x0A2C2A** [comment]
yes because who is actually going to buy shiny

**Comment by TheGreenSquare** [comment]
> yes because who is actually going to buy shiny

idiots in arrascord with more money than sense

**Comment by 0x0A2C2A** [comment]
> > yes because who is actually going to buy shiny
> 
> idiots in arrascord with more money than sense

real

**Comment by KerbalBoy** [comment]
Hey, what the hell?

**Comment by TheGreenSquare** [comment]
the $3 shiny was actually kinda worth it back in the day before they nerfed it and made the funny tanks $10 a month
